source/content.css

```diff
@@ -506,6 +506,12 @@ div.inline-comment-form .form-actions,
 	padding: 6px 9px;
 }
 
+/* Decrease the padding of filters dropdown in PR filters to accommodate new filter - Filter by build status */
+.table-list-filters .table-list-header-toggle .select-menu-button {
+	padding-left: 10px;
+	padding-right: 10px;
+}
```

Comment:
This is changed somewhere else in the CSS already, make sure to reuse that one otherwise it’s duplicate.

However I don’t think it’s still enough since your screenshot is missing “Open all” which would fill all the remaining space anyway

Reply:
Thanks! I looked into it and found one place. When we have the `Open all` added before the filters, the padding is being reduced to `12px`. It is only for the case when `Open all` is added. So to prevent the overflow issue I have reduced that padding to `8px`. Which doesn't look bad, IMO. 

![Screen Shot 2019-04-17 at 12 09 58 AM](https://user-images.githubusercontent.com/22439276/56237504-c3123280-60a9-11e9-8da4-68835dd0506e.png)

So when the `Open all` is present the padding will be `8px` and when it's absent the padding will be `10px`.